### ISSUE TITLE
Change help.py 

### ISSUE THREAD

**Issue by Puncher1** [School-Simplified/Timmy-SchoolSimplified#229]
To a selfmade and simple solution with a select menu.

**Comment by Puncher1** [comment]
I'm here to help

**Comment by pandabear189** [comment]
> To a selfmade and simple solution with a select menu.

Do you mean like waiting for dpy cls `commands.HelpCommand` to include application commands? Currently, subclassing would create more problems, the current cog cls mixes *methods from RoboDanny, ext.commands.help, and app_commands

**Comment by Puncher1** [comment]
Yeah we have to wait until it's implemented. After that we can create our own help command. So it's not high priority, the current solution works for the moment.
